operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Configuration.java

```diff
@@ -39,6 +39,9 @@
       + "Legal values are DYNAMIC (the default) and ON_RESTART. See also introspectVersion.")
   private OverrideDistributionStrategy overrideDistributionStrategy;
 
+  @Description("Configuration for istio property.")
```

Comment:
"Istio service mesh integration" or something like that

Reply:
The Istio service mesh integration settings